src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -286,12 +286,12 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
   confirmArticles = false
   isEditingType = false
   dropdown: boolean = null
-  supportedEntityTypes = null
+  supportedEntityTypes: Array<string> = []
 
   /** Called when component is mounted. */
   mounted (): void {
     this.initializeEntityType()
-    this.supportedEntityTypes = GetFeatureFlag('supported-alteration-change-business-types') || []
+    this.supportedEntityTypes = GetFeatureFlag('supported-alteration-change-business-types')
```

Comment:
I would still fall back to `[]` in case LD returns null/undefined.

Reply:
Hahah I had a feeling - ok adding it to the list. We're going to have to make some more changes for the payment codes I believe

Reply:
It's the conditional chaining on line 325 that you don't need if this variable is always an array (not falsy).